sdk/web-pubsub/web-pubsub/src/hubClient.ts

```diff
@@ -266,6 +271,13 @@ export interface GenerateClientTokenOptions extends OperationOptions {
    * The groups to join when the client connects
    */
   groups?: string[];
+
+  /**
+   * The endpoint type of the client
+   * * `default`: Default WebPubSub Client. Example Service URL: _wss://exampleHost.com/client/hubs/exampleHub_
+   * * `mqtt`: MQTT Client. Example Service URL:  _wss://exampleHost.com/client/mqtt/hubs/exampleHub_
+   */
+  webPubSubClientProtocol?: WebPubSubClientProtocol;
```

Comment:
```suggestion
 clientProtocol?: WebPubSubClientProtocol;
```

My thinking is that these being options to creating a client, using the prefix webPubSub is likely redundant - is there a particular reason we decided to keep the prefix?

note: i am referring to the parameter name here, _not_ the parameter"s type. is there a reason we decided not to go with:

```ts
clientProtocol?: WebPubSubClientProtocol;
```

Reply:
Actually, you're right. Let me change it to clientProtocol instead

Reply:
yeah you're right, let me remove it then